packages/lib-user/src/hooks/usePanoptesUserGroup.js

```diff
@@ -29,6 +29,6 @@ async function fetchPanoptesUserGroup({ groupId, token }) {
 
 export function usePanoptesUserGroup({ adminMode, authUserId, groupId, membershipId }) {
   const token = usePanoptesAuthToken()
-  const key = groupId ? { groupId, token } : null
+  const key = groupId ? { adminMode, groupId, token, membershipId } : null
```

Comment:
This refactor re-requests a group if a user toggles admin mode or after a user joins/leaves a group and gets an updated group membership ID. Currently after joining a group the success toast will show but the group will not be loaded until SWR is triggered to re-request the group.